### PR TITLE
Add logout button stories, add logoutt functionality

### PR DESCRIPTION
**Pull request by Polo11121**

### CODE REVIEW

File: src/stories/Button/Button.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import logout from '../../actions/logout';
```
Comment (Polo11121) [inline src/stories/Button/Button.stories.tsx:3]: usless import

File: src/context/AuthProvider.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const AuthProvider: React.FC<AuthProviderProps> = ({ children }) => {
   useEffect(() => {
     auth.onAuthStateChanged(user => {
       if (user) setCurrentUser(user);
```
Comment (MateuszLisowski) [inline src/context/AuthProvider.tsx:16]: please add curly braces for if and else. 
Please also read https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces 
I think not using {} has some drawbacks

File: src/actions/logout.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useHistory } from 'react-router-dom';
+import { auth } from '../firebase';
+
+const history = useHistory();
+
+const logout = () =>
+  auth
+    .signOut()
+    .then(() => history.push('/'))
+    .catch(error => alert(error.message));
```
Comment (MateuszLisowski) [inline src/actions/logout.tsx:10]: I think in catch we shouldn't use alert, I've never seen this kind of method for error handling. I think the best way for now is console.error or console.log wirh error.message
Or we could build global error boundary (something like context wrapper but for catching the errors which could be thrown in catch)

**Review by MateuszLisowski — CHANGES_REQUESTED** [review]

File: src/stories/Button/Button.stories.tsx (hunk 1 above)
Comment (MateuszLisowski) [reply src/stories/Button/Button.stories.tsx:3]: So why eslint allows you to push it? You should add "no-useless-import" rule or something like that

**Review by MateuszLisowski — APPROVED** [review]

**Pull request merged**